### PR TITLE
Theme Patch🚀 : New theme added in theme.js and new Theme attributes added.

### PR DESCRIPTION
**Pull request by tamojit-123**
Added new Themes and added new theme attributes 

- [x] yellowTheme 
- [x] materialDarkTheme
- [x] materialLightTheme 
- [x] materialTealTheme
- [x] screenshot

### CODE REVIEW

**Comment by saiteja13427** [comment]
@tamojit-123 

1. Can you please remove package-lock.json from the PR.
2. Also any specific reason of adding expTxtColor and splashBg?

**Comment by tamojit-123** [comment]
> @tamojit-123 You are not supposed to delete the package-lock.json
> 
> You can follow this guide https://blog.devgenius.io/how-to-remove-modified-or-changed-files-from-a-git-pull-request-a7769e3c58da to get the package-lock to the state it is in master and make a commit. This will remove it from this PR.

Hi @saiteja13427 and @ashutosh1919 all checks are passed

**Comment by saiteja13427** [comment]
@tamojit-123 Awesome, I will once check these themes on my local system and merge it.

**Comment by saiteja13427** [comment]
@tamojit-123 I have checked the themes, there are a few improvements to be made
1. **MaterialDarkTheme**: The experience card text colour is black so it is not properly visible
2. **MaterialLightTheme**: I think we can try tweaking it a bit as the svgs doesn't look that great. For example, the below one has all white clothes which doesn't look that great I think. Same with a few other svgs as well.
image
3. **MaterialTealTheme**: Same as the point 2.

Let me know your views on it.

**Comment by tamojit-123** [comment]
@saiteja13427 I will work on it I know the issue

**Comment by tamojit-123** [comment]
Hi @saiteja13427 please check now I made some tweek

File: src/theme.js [hunk 1]
```diff
@@ -2,121 +2,197 @@
...
+  secondaryText: "#05505E",
+  imageHighlight: "#0a343c",
+  compImgHighlight: "#E1E2E1",
+  jacketColor: "#E1E2E1",
+  headerColor: "#E1E2E1",
+  splashBg: "#05505E",
 };
 
-export const chosenTheme = blueTheme;
+export const chosenTheme = materialDarkTheme;
```
Comment (saiteja13427) [inline src/theme.js:198]: Let us keep the default theme as blueTheme only!

File: src/containers/experienceAccordion/ExperienceAccordion.js [hunk 2]
```diff
@@ -24,6 +24,7 @@ class ExperienceAccordion extends Component {
                       borderColor: `${theme.headerColor}`,
                       marginBottom: `3px`,
                       fontFamily: "Google Sans Regular",
+                      color: `${theme.text}`,
```
Comment (saiteja13427) [inline src/containers/experienceAccordion/ExperienceAccordion.js:27]: Let us add a hover colour as well. Currently on hover text turns black which again looks odd, so let us add hover as well.

Just add 
```
':hover': {
    color: `${theme.secondaryText}`
}
```

**Comment by saiteja13427** [comment]
@tamojit-123 Just two small changes and we are good to go.

File: src/containers/experienceAccordion/ExperienceAccordion.js (hunk 2 above)
Comment (tamojit-123) [reply src/containers/experienceAccordion/ExperienceAccordion.js:27]: Fixed : https://github.com/ashutosh1919/masterPortfolio/pull/253/commits/b3c3cf5742c1c6846cdcf577b1903fe8aeaf07df

File: src/theme.js (hunk 1 above)
Comment (tamojit-123) [reply src/theme.js]: Fixed : https://github.com/ashutosh1919/masterPortfolio/pull/253/commits/b3c3cf5742c1c6846cdcf577b1903fe8aeaf07df

**Comment by tamojit-123** [comment]
Hi @saiteja13427 the fixes are done: https://github.com/ashutosh1919/masterPortfolio/pull/253/commits/b3c3cf5742c1c6846cdcf577b1903fe8aeaf07df
please review once

**Review by saiteja13427 — APPROVED** [review]
Looks Good!

@ashutosh1919 Have a look once. I will merge it once you have a look.

**Comment by ashutosh1919** [comment]
@tamojit-123 or @saiteja13427 can one of you please post screenshots of the UI here for all the new themes added? I just want to see how it looks

**Comment by tamojit-123** [comment]
# material dark theme
Screenshot (4)
Screenshot (5)
Screenshot (6)
Screenshot (7)
Screenshot (8)
Screenshot (9)
Screenshot (10)
Screenshot (11)
Screenshot (12)
Screenshot (13)
Screenshot (14)
Screenshot (15)
Screenshot (16)
Screenshot (17)
Screenshot (18)
Screenshot (19)

**Comment by tamojit-123** [comment]
# material light theme
Screenshot (20)
Screenshot (21)
Screenshot (22)
Screenshot (23)
Screenshot (24)
Screenshot (25)
Screenshot (26)
Screenshot (27)
Screenshot (28)
Screenshot (29)
Screenshot (30)
Screenshot (31)
Screenshot (32)

**Comment by tamojit-123** [comment]
# material teal theme
Screenshot (33)
Screenshot (34)
Screenshot (35)
Screenshot (36)
Screenshot (37)
Screenshot (38)

**Comment by tamojit-123** [comment]
# yellow theme
Screenshot (39)
Screenshot (40)
Screenshot (41)
Screenshot (42)
Screenshot (43)
Screenshot (44)

**Comment by tamojit-123** [comment]
Hi, @ashutosh1919 and @saiteja13427 I attached all the theme look.

**Review by ashutosh1919 — APPROVED** [review]
LGTM! @saiteja13427 please double check the changes and merge if it looks good to you

**Comment by saiteja13427** [comment]
Checked again! Looks Good. Merging it.

Good work @tamojit-123. Thanks for the contribution and all the specified changes.

Keep contributing!!

**Comment by saiteja13427** [comment]
@all-contributors please add @tamojit-123 for code and design

**Comment by tamojit-123** [comment]
> Checked again! Looks Good. Merging it.
> 
> Good work @tamojit-123. Thanks for the contribution and all the specified changes.
> 
> Keep contributing!!

Thank you @saiteja13427 for your support

**Pull request merged**